docs/2-fork-accountability.md

```diff
@@ -0,0 +1,161 @@
+# Fork accountability
+
+As we know, the agreement property of the Tendermint consensus algorithm cannot be violated if there are at most *f* faulty processes in the system.
+What happen if more than *f* processes are faulty?
+
+A well-designed consensus protocol should provide some guarantees when this limit is exceeded. The most important guarantee is **fork accountability**, where the processes that caused the consensus to fail can be identified and punished according to the protocol specifications. 
+
+## The problem
+
+We can separate two cases:
+
+- if the number of faulty processes is more than *2f*, we cannot do anything because faulty processes basically control the system and they can do whatever they prefer.
```

Comment:
When we define a problem called "Accountability" or smething like that, we will say here that this problem can not be solved when there are more than 2f faulty processes.
However, it really can be solved with piggybacking justifications in every single message.
I think we should avoid mentioning this case.
If we still want to mention it, we would need to mention this "third" way of achieving accountability.

Reply:
I agree, I removed that case and just mentioned the case where the problem is addressed.